gcp-native-ts-k8s-ruby-on-rails-postgresql/README.md

```diff
@@ -0,0 +1,120 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/gcp-ts-k8s-ruby-on-rails-postgresql/infra)
+
+# Containerized Ruby on Rails App Delivery on GCP Native
+
+This example is a full end to end example of delivering a containerized Ruby on Rails application. It
+
+-   Provisions a [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/) cluster
+-   Provisions a fully managed Google Cloud SQL PostgreSQL database
+-   Builds a containerized Ruby on Rails container image, and publishes it to Docker Hub
+-   Deploys that container image as a Kubernetes Service inside of the provisioned GKE cluster
+
+All of these happen behind a single `pulumi up` command, and are expressed in just a handful of TypeScript.
+
+## Prerequisites
+
+Ensure you have [downloaded and installed the Pulumi CLI](https://www.pulumi.com/docs/get-started/install/).
+Ensure you have [downloaded and installed Docker](https://docs.docker.com/install/)
+We will be deploying to Google Cloud Platform (GCP), so you will need an account. If you don't have an account,
+[sign up for free here](https://cloud.google.com/free/). In either case,
+[follow the instructions here](https://www.pulumi.com/docs/intro/cloud-providers/gcp/setup/) to connect Pulumi to your GCP account.
+
+This example assumes that you have GCP's `gcloud` CLI on your path. This is installed as part of the
+[GCP SDK](https://cloud.google.com/sdk/).
+
+## Running the Example
+
+After cloning this repo, `cd infra/` and run these commands. After 8 minutes, you'll have a fully functioning GKE
+cluster and containerized Ruby on Rails application deployed into it, using a hosted PostgreSQL instance!
+
+1. Create a new stack, which is an isolated deployment target for this example:
+
+    ```bash
+    $ pulumi stack init gcp-rails-dev
+    ```
+
+2. Set the required configuration variables for this program:
+
+    ```bash
+    $ pulumi config set gcp:project [your-gcp-project-here]
+    $ pulumi config set gcp:region us-west1 # any valid region
+    $ pulumi config set gcp:zone us-west1-a # any valid GCP zone under above region
+    $ pulumi config set clusterPassword --secret [your-new-cluster-password-here] # must be at least 16 characters
+    $ pulumi config set dbUsername [your-new-db-username-here]
+    $ pulumi config set dbPassword --secret [your-new-db-password-here]
```

Comment:
Are these two required? Let's hard/code and/or generate them.

Reply:
Done.